src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java

```diff
@@ -56,23 +56,32 @@ public interface InvoiceUserApi extends KillbillApi {
      * @param accountId account id
      * @param includesMigrated flag that indicates if migrated invoices should be returned
      * @param includeVoidedInvoices flag that indicates if voided invoices should be returned
-     * @param withInvoiceItems flag that indicates if invoice items should be returned
+     * @param includeInvoiceComponents flag that indicates if invoice components (items, payments, etc.) should be returned
      * @param context   the tenant context
      * @return all invoices
      */
-    public List<Invoice> getInvoicesByAccount(UUID accountId, boolean includesMigrated, boolean includeVoidedInvoices, boolean withInvoiceItems, TenantContext context);
+    public List<Invoice> getInvoicesByAccount(UUID accountId, boolean includesMigrated, boolean includeVoidedInvoices, boolean includeInvoiceComponents, TenantContext context);
+    
+    /**
+     * @param accountId account id
+     * @param offset the offset of the first result
+     * @param limit the maximum number of results to retrieve
+     * @param context the tenant context
+     * @return paginated invoices
+     */
+    public Pagination<Invoice> getInvoicesByAccountWithPagination(UUID accountId, Long offset, Long limit, TenantContext context);
```

Comment:
@sbrossie Naming is hard... Any thought? What about `getPaginatedInvoicesByAccount`? or `listInvoicesByAccount` (mirrors the `search` APIs)?

Reply:
What about:

```
    public Pagination<Invoice> getInvoicesByAccount(UUID accountId, Long offset, Long limit, TenantContext context);
```

(The method signature is different so it works, and we already have multiple `getInvoicesByAccount` so this is consistent with adding one more.

Reply:
👍